app/templates/manage_users.html

```diff
@@ -74,4 +74,5 @@ <h5 class="modal-title" id="userModalLabel">Add/Edit User</h5>
     let users = JSON.parse('{{ users | tojson | safe }}');
   </script>
   <script src="{{ url_for('static', filename='js/manage_users.js') }}"></script>
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/manage-users.css') }}"/>
```

Comment:
Can you move this to the top after the `<h2>` tag? CSS should be loaded before rendering the elements, and JavaScript should be run post-rendering.